e2e/run/run.go

```diff
@@ -266,6 +266,134 @@ func (c ctx) testFuseSquashMount(t *testing.T) {
 	)
 }
 
+// testRunOCICDI checks that passing --device and --cdi-dirs flags to run
+// with an OCI/ORAS image work without causing a crash or hard failure.
+// Apptainer doesn't yet implement full CDI injection on the native
+// launcher; this test primarily ensures the CLI accepts the flags and
+// the image can still be pulled/run successfully.
+func (c ctx) testRunOCICDI(t *testing.T) {
```

Comment:
Why is "OCI" in the name of this function?  I think that may just be a leftover from Sylabs' implementation in only oci mode.